Data/Vector/Generic/Mutable.hs

```diff
@@ -13,7 +13,7 @@
 
 module Data.Vector.Generic.Mutable (
   -- * Class of mutable vector types
-  MVector(..),
+  MVector(..), PrimMonad(..), RealWorld,
```

Comment:
It's a bit weird to export `PrimMonad(..)` / `RealWorld` under "Class of mutable vector types", and render almost at the very top of haddocks. What about moving them to a new section "Re-exported entities" at the bottom of export list?

Reply:
Good point. I moved both to bottom of export list